README.md

```diff
@@ -1 +1,13 @@
 ## Cloud integration in Home Assistant
+
+
+
+## Updating voice data
+
+To update the voice data with the latest from Azure, run the following script:
+
+```
+python3 -m script.update_voice_data
+```
+
+You will need to fetch an Azure TTS token. You can generate one by running the [sample key generator server](https://github.com/Azure-Samples/cognitive-services-speech-sdk/tree/master/samples/js/browser/server) and visiting `http://localhost:3001/api/get-speech-token`.
```

Comment:
Looking at that repo, it seems like this require another token, and that token is used to generate this.
Would it then not be better to just generate this on the fly in the `update_voice_data` script?

And is this really something that are viable?
It says it is a trial of 30 days.
I think a better approach here, is that we are not making this one "contributable", but instead add a action that runs weekly to generate this?

Reply:
Yeah, that would be fine. Even monthly could work, I don't expect it to change much. However, not something I am able to do.

The link to the repository has a readme that explains how to get the key.